### DISCUSSION TITLE
Container keeps connecting to google.com

### DISCUSSION THREAD

**Discussion by devopsbarbarian** [haugene/docker-transmission-openvpn#1977] [Q&A] [answered]
Hi! In Pihole, it caught my attention that transmission-openvpn keeps pinging google.com 4 times a second, all the time. What's the cause of this and can we prevent it?

I use Mullvad as my vpn provider.

**Comment by pkishino** [comment, marked as answer]
Check your healthcheck setting for the container.. the default check is Google.com but 4 times a second isn’t right..

**Reply by devopsbarbarian** [reply]
Strange, I inspected the container and healthcheck seems to be set up properly:
`HEALTHCHECK &{["CMD-SHELL" "/etc/scripts/healthcheck.sh"] "1m0s" "0s" "0s" '\x00'}`
This doesn't seem to work for me, it checks every second or so (seems to be 4 hits per healthcheck actually). When I manually set the interval to 1m in compose, it fixes the issue.

**Reply by pkishino** [reply]
can you show me the config part that fixed this please? 
I'll see if I can narrow this down, perhaps something not correctly scripted
